### ISSUE TITLE
which cuda and cudnn version did you use?

### ISSUE THREAD

**Issue by byuns9334** [JasonSWFu/MetricGAN#4]
when I run ./MetricGAN.py, I get some number (maybe loss) and error, just like this:
How do I fix this error? which cuda & cudnn versions did you use? (The keras, librosa, python versions are exactly same with yours)

0.765720826757873
2.1266
/home/users/woody/vocoder/speech_enhancement/metricGAN/local/lib/python2.7/site-packages/matplotlib/axes/_base.py:3152: UserWarning: Attempting to set identical left==right results
in singular transformations; automatically expanding.
left=1, right=1
  'left=%s, right=%s') % (left, right))
Sample training data for discriminator training...
Discriminator training...
/home/users/woody/vocoder/speech_enhancement/metricGAN/local/lib/python2.7/site-packages/keras/engine/training.py:973: UserWarning: Discrepancy between trainable weights and collected trainable weights, did you set `model.trainable` without calling `model.compile` after ?
  'Discrepancy between trainable weights and collected trainable'
Epoch 1/1
2019-09-03 18:42:10.063259: E tensorflow/stream_executor/cuda/cuda_dnn.cc:403] could not create cudnn handle: CUDNN_STATUS_NOT_INITIALIZED
2019-09-03 18:42:10.063335: E tensorflow/stream_executor/cuda/cuda_dnn.cc:411] possibly insufficient driver version: 384.130.0
2019-09-03 18:42:10.063354: F tensorflow/core/kernels/conv_ops.cc:712] Check failed: stream->parent()->GetConvolveAlgorithms( conv_parameters.ShouldIncludeWinogradNonfusedAlgo<T>(), &algorithms)
Aborted (core dumped)

**Comment by byuns9334** [comment]
@JasonSWFu

**Comment by JasonSWFu** [comment]
Sorry, my server is out of work now, so I cannot get the related information. Try to update your GPU driver version?

**Comment by byuns9334** [comment]
@JasonSWFu Is your server recovered?
